system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -167,7 +169,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("bridge-hub-polkadot"),
 	impl_name: create_runtime_str!("bridge-hub-polkadot"),
 	authoring_version: 1,
-	spec_version: 1_002_000,
+	spec_version: 1_002_001,
```

Comment:
Don't know if I need to do it here, or it'll be done on release. But it also need to be changed for Kusama BH as well, as we are going to apply the same patch there

Reply:
like you said, https://github.com/polkadot-fellows/runtimes/releases/tag/v1.2.1 is already used for people chain runtime release - let's do 1.2.2 for BHs patch release (same for Kusama)

```suggestion
	spec_version: 1_002_002,
```

Reply:
Ok, I've added similar bump to Kusama BH too